extension/content_script.js

```diff
@@ -257,6 +257,7 @@ function unhighlightEverything() {
 	}
 	$("." + keywordClass).unmark();
 	highlightedSentenceId = null;
+	$(".persistentHighlight").removeClass("persistentHighlight");
```

Comment:
Please refactor all "persistentHighlight" string into a variable (I guess it has to be global here)
-- In general if we see a hardcoded number / string value, make a variable so it's only declared once.
Some benefits
- If someday you want to change it, it's just in one place.
- If you mistype a string, you won't get a complaint. If you mistype a variable, thanks to strict mode, you will get a complaint.

-- Better to do it now before the usage spreads everywhere